src/main/java/uk/gov/hmcts/reform/idam/web/AppController.java

```diff
@@ -449,6 +449,9 @@ public String forgotPassword(@ModelAttribute("forgotPasswordCommand") @Validated
                     forgotPasswordRequest.getEmail(),
                     forgotPasswordRequest.getRedirectUri(),
                     forgotPasswordRequest.getClientId());
+
+                model.put(SELF_REGISTRATION_ENABLED, isSelfRegistrationEnabled(forgotPasswordRequest.getClientId()));
```

Comment:
If the request is coming from Web Admin, the clientId will be `null` - could that cause any issues?

Reply:
It doesn't create any problem, but I think it is better to save a call to api and FR if client id is null.